lib/gui/utils/byte-size/byte-size.js

```diff
@@ -26,6 +26,11 @@
 const angular = require('angular');
 const MODULE_NAME = 'Etcher.Utils.ByteSize';
 const ByteSize = angular.module(MODULE_NAME, []);
+
+/* eslint-disable lodash/prefer-lodash-method */
```

Comment:
Personally, I agree that being strict with exceptions, is better than being lax and allowing accidental inconsistencies.